lib/pages/main/wallet_contents/transfer_asset.dart

```diff
@@ -645,6 +647,12 @@ class _TransferAssetState extends State<TransferAsset> {
       return;
     }
```

Comment:
can we display the error message below the field as other validations instead of the snackbar?

Reply:
I did a workaround as the text field validations are known to be only sync, please try it.